lib/display.ex

```diff
@@ -31,10 +32,39 @@ defmodule Display do
     IO.puts(format_red(Exception.format(:error, error, trace)))
   end
 
-  defp line_number({_, [line: line], _}) do
-    line
+  defp line_number({_, [line: line], _}, in: _), do: line
+  defp line_number(expr, in: file) do
```

Comment:
When doing `assert_receive <expr>` all I get passed in as the first argument to `line_number` is literally `<expr>`. I don't know if I have an error before that gets even called?